media-autobuild_suite.bat

```diff
@@ -688,6 +688,26 @@ if %buildffmpegUp%==3 set "ffmpegUpdate=onlyFFmpeg"
 if %buildffmpegUp% GTR 3 GOTO ffmpegUp
 if %deleteINI%==1 echo.ffmpegUpdate=^%buildffmpegUp%>>%ini%
 
+:ffmpegVer
+if %ffmpegVersionINI%==0 (
+    echo -------------------------------------------------------------------------------
+    echo -------------------------------------------------------------------------------
+    echo.
+    echo. Always build latest FFmpeg?
+    echo. 1 = Yes
+    echo. x.y.z = Specific FFmpeg version (the git tag, e.g. 4.2.2 for n4.2.2^)
```

Comment:
No, this seems like a terrible ideia.
There's no verification at all of what the user writes.

Reply:
No guarantee is needed. `do_vcs` will fail and log will be written.

Reply:
Yeah, the user does stupid things and we have to deal with it. That's not how it works.